core-bundle/contao/templates/twig/content_element/text.html.twig

```diff
@@ -1,14 +1,16 @@
 {% extends "@Contao/content_element/_base.html.twig" %}
 {% use "@Contao/component/_figure.html.twig" %}
 
-{% set content_layout_attributes = attrs(content_layout_attributes|default)
+{% set content_layout_attributes = attrs()
     .addClass('media media--' ~ layout, layout and image)
+    .mergeWith(content_layout_attributes|default)
 %}
 
 {# By default we add the layout attributes to the wrapper element. If you want to
  disable this behavior, set "default_content_layout" to false. #}
-{% set attributes = attrs(attributes|default)
+{% set attributes = attrs()
     .mergeWith(content_layout_attributes, default_content_layout is not defined or default_content_layout)
+    .mergeWith(attributes|default)
 %}
```

Comment:
I think in this case we actually want the `content_layout_attributes` to take precedence. So I’d keep this as it was before.

Reply:
Reverted in cecd9589521216ad5334cf7a71c382ea42e5baad.